package.json

```diff
@@ -21,7 +21,6 @@
     "@stripe/react-stripe-js": "^1.4.0",
     "@stripe/stripe-js": "^1.13.2",
     "antd": "^4.15.0",
-    "axios": "^0.21.1",
```

Comment:
I'd be careful eliminating this dependency. I would assume most network requests are being made with this which would break a lot of the application. Be sure you really really mean to get rid of this

Reply:
Axios is reinstalled